feeds-lookup.js

```diff
@@ -273,19 +273,21 @@ exports.init = function (sbot, config) {
         live = true,
         old = false,
         root = null,
-        tombstoned = false,
+        tombstoned = null,
       } = opts || {}
 
       const filterRootFn = root
         ? (branch) => branch.length > 0 && branch[0][0] === root
         : () => true
 
       const filterTombstoneOrNot = (branch) => {
```

Comment:
I think this function is one of those cases where I would maybe add a comment to each branch to explain why it is correct